docs/lms-connection.md

```diff
@@ -76,18 +75,18 @@ LMS_BACKENDS=[
     {
         "BACKEND": "richie.apps.courses.lms.edx.TokenEdXLMSBackend",
         "SELECTOR_REGEX": r".*lms-example1.org.*",
+        "JS_SELECTOR_REGEX": r".*lms-example1.org.*",
         "BASE_URL": "https://www.lms-example1.org",
-        "COURSE_REGEX": r"^.*/courses/(?P<course_id>.*)/info$",
-        "API_TOKEN": "fakesecret1",
+        "JS_COURSE_REGEX": r"^.*/courses/(?<course_id>.*)/info$",
     },
     {
         "BACKEND": "richie.apps.courses.lms.edx.TokenEdXLMSBackend",
         "SELECTOR_REGEX": r".*lms-example2.org.*",
+        "JS_SELECTOR_REGEX": r".*lms-example2.org.*",
         "BASE_URL": "https://www.lms-example2.org",
-        "COURSE_REGEX": r"^.*/course/(?P<course_id>[0-9]*)$",
-        "API_TOKEN": "fakesecret2",
+        "JS_COURSE_REGEX": r"^.*/course/(?<course_id>[0-9]*)$",
```

Comment:
why JS_* ? You could continue calling them COURSE_REGEX and SELECTOR_REGEX and pass them to javascript?

Reply:
Because I need a SELECTOR_REGEXP for LMS_Handler in python code. So I prefixed all regexp used in JS with JS_ to distinguish them.

Reply:
But they have the same value... Could they ever be different?

Reply:
Nope this is not the same value. There is a little difference about how to declare group.
Python syntax is `?P<group_name>` and in JS syntax is `?<group_name>`